src/get-started/npm-script.md

```diff
@@ -3,6 +3,38 @@ layout: page
 title: Get Started > NPM Script
 ---
 
-# NPM Script
+# NPM
 
-Need to write up.
+If you already have a build process based on npm scripts you can use `workbox-
+cli` to generate a service worker.
+
+1. Install the module with NPM.
+
+    ```
+    npm install --save-dev workbox-cli
+    ```
+
+1. From your project folder, run `workbox-cli` from a command line. A wizard
```

Comment:
Your previous step correctly instructs developers to run

```
npm install --save-dev workbox-cli
```

This won't add `workbox-cli` to the global $PATH, though, so you can't just run `workbox-cli` from the command line. It won't be found. It's installed locally in `node_modules/.bin/workbox-cli`. (This is just how `npm` works, and I don't think it's worth getting into in your docs!)

What I'd recommend is switching the order of this point and the second point. So tell the developer to modify their `build` command first, and then after they do that, in the second step, tell them that the first time they `npm run build`, they'll be asked some questions and those answers will be reused.